xmtp_mls/migrations/2024-12-18-170645_add_dm_id/up.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE groups ADD COLUMN dm_id TEXT;
+ALTER TABLE groups ADD COLUMN last_message_ns BIGINT NOT NULL DEFAULT ((strftime('%s', 'now') * 1000000000) + (strftime('%f', 'now') * 1000000));
```

Comment:
Is this going to conflict with the work @mchenani is doing for last message performance?

Reply:
Yes, it will!